templates/reverse-proxy-auth-jenkins-config.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<hudson>
+  <useSecurity>true</useSecurity>
+  <numExecutors>0</numExecutors>
+  <authorizationStrategy class="hudson.security.FullControlOnceLoggedInAuthorizationStrategy">
+    <denyAnonymousReadAccess>true</denyAnonymousReadAccess>
+  </authorizationStrategy>
+  <securityRealm class="org.jenkinsci.plugins.reverse_proxy_auth.ReverseProxySecurityRealm" plugin="reverse-proxy-auth-plugin@1.7.7">
+    <managerPasswordSecret>{AQAAABAAAAAQPt9fH9Ky04iMhNChDOcoYRX3maoyWTClSy2MTqXT850=}</managerPasswordSecret>
```

Comment:
I'm wondering if this is expected?

Reply:
Good catch. This is expected to be populated every time jenkins is restarted, so we can just remove the line :)